modules/local/sum_taxonomy.nf

```diff
@@ -31,15 +31,14 @@ process SUM_TAXONOMY {
     library(readr)
     library(tidyr)
     library(stringr)
+    library(tidyverse)
```

Comment:
```suggestion
    library(purrr)
```
I'm not sure which packages you actually need, but I think `read_df()` is from `purrr`.